frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -549,19 +557,33 @@ function useMathSelectorOptions({
         mathAvailability != MathAvailability.ActorsOnly ? staticMathDefinitions : staticActorsOnlyMathDefinitions
     )
         .filter(([key]) => {
-            if (!isStickiness) {
-                return true
+            if (isStickiness) {
+                // Remove WAU and MAU from stickiness insights
+                return !TRAILING_MATH_TYPES.has(key)
+            }
+            return true
+        })
+        .map(([key, definition]) => {
+            const shouldWarnAboutTrailingMath =
+                TRAILING_MATH_TYPES.has(key) && trendsDisplayCategory === ChartDisplayCategory.TotalValue
+            return {
+                value: key,
+                icon: shouldWarnAboutTrailingMath ? <IconWarning /> : undefined,
+                label: definition.name,
+                tooltip: !shouldWarnAboutTrailingMath ? (
+                    definition.description
+                ) : (
+                    <>
+                        <p>{definition.description}</p>
+                        <i>
+                            In total value insights, it's usually not clear what date range "{definition.name}" refers
+                            to. For full clarity, we recommend using "Unique users" here instead
```

Comment:
I reworded this a bit to hopefully suggest "Unique users" more gently @thmsobrmlr The tricky bit is that it's really hard to understate how confusing MAU/WAU is in this situation 😅